draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -279,6 +279,109 @@ etc., are all valid and equivalent header field names. However, case is signific
 
 ## Option 1: DPoP-Inspired Authentication {#dpop-esque-auth}
 
+This option, inspired by the OAuth DPoP specification {{?RFC9449}}, uses a DPoP-like mechanism to authenticate
+the calling workload in the context of the request. The WIMSE Identity Token {{to-wit}} is sent in the request as
+described in {{wit-http-header}}. An additional JWT, the Workload Proof Token (WPT), is signed by the private key
+corresponding to the public key in the WIT. The WPT is sent in the `Workload-Proof-Token` header field of the request.
+A WPT contains the following:
+
+* in the JOSE header:
+    * `alg`: An identifier for an appropriate JWS asymmetric digital signature algorithm corresponding to
+     the confirmation key in the associated WIT.
+    * `typ`: the WPT is explicitly typed, as recommended in {{Section 3.11 of RFC8725}},
+     using the `application/wimse-proof+jwt` media type.
+* in the JWT claims:
+    * `iss`: The issuer of the token, which is the calling workload, represented by the same value as the `sub` claim
+     of the associated WIT.
```

Comment:
You might want to also include the hash of the WIT in the proof of possession.  Perhaps just having the full name is good enough, but I think you want to account for the issuer of the WIT as well in the signature.

Reply:
> You might want to also include the hash of the WIT in the proof of possession. 

What purpose would that serve? Asking for a friend. 

> Perhaps just having the full name is good enough, but I think you want to account for the issuer of the WIT as well in the signature.

I don't think I follow the thinking here...

Reply:
action from the call was to put in an issue for further discussion and proceed without a WIT hash for now

Reply:
issue here https://github.com/yaronf/wimse-s2s/issues/24 for posterity and resolving this conversation